cmd/flowrunner/testdata/flows/all_actions_test.json

```diff
@@ -1,4 +1,5 @@
 {
+  "caller_events": [],
```

Comment:
ah i forgot they were previously called "resume_events" before I renamed them to "caller_events"